lib/Db/SynchronizationMapper.php

```diff
@@ -73,12 +73,11 @@ public function findAll(?int $limit = null, ?int $offset = null, ?array $filters
 
 	public function createFromArray(array $object): Synchronization
 	{
+		$object['uuid'] = Uuid::v4();
+
 		$obj = new Synchronization();
 		$obj->hydrate(object: $object);
-		// Set uuid
-		if ($obj->getUuid() === null){
```

Comment:
Why remove this?
And if on purpose for a good reason, should we not do it for all mappers with the same if statement?